manifests/z/ZAP/ZAP/2.13.0/ZAP.ZAP.installer.yaml

```diff
@@ -0,0 +1,21 @@
+PackageIdentifier: ZAP.ZAP
+PackageVersion: 2.13.0
+InstallerType: exe
+InstallModes:
+- silent
+InstallerSwitches:
+  Silent: -q
+  SilentWithProgress: -q
+UpgradeBehavior: install
+ReleaseDate: 2023-07-12
```

Comment:
```suggestion
ReleaseDate: 2023-07-12
ElevationRequirement: elevationRequired
```


> ElevationRequirement - Indicator for elevation requirements when installing or upgrading packages.
>
> Optional Field
>
> This key represents which scope a package is required to be executed under. Some packages require user level execution while others require administrative level execution.
>
> * elevationRequired - Must be run from a shell that is running in an administrative context (e.g - Admin user using powershell/terminal/cmd with "Run as Administrator")
> * elevationProhibited - Must be run from a shell that is not running in an administrative context
> * elevatesSelf - If called from a non-administrative context, will request elevation. If called from an administrative context, may or may not request elevation.

Reply:
Where are those docs?

Reply:
Found it https://github.com/microsoft/winget-pkgs/blob/master/doc/manifest/schema/1.4.0/installer.md :)